internal/attestation/crafter/crafter.go

```diff
@@ -43,6 +43,28 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+// attestationCraftingMaterial is a map of material types to their corresponding schemaapi.CraftingSchema_Material_MaterialType
+var attestationCraftingMaterial = func() map[string]schemaapi.CraftingSchema_Material_MaterialType {
```

Comment:
why do you need to return `schemaapi.CraftingSchema_Material_MaterialType`? Is it using anywhere?

Reply:
Moved to `schemaapi` package.